OpenCore-Boot.sh

```diff
@@ -62,6 +62,7 @@ args=(
   # -netdev user,id=net0 -device vmxnet3,netdev=net0,id=net0,mac=52:54:00:c9:18:27  # Note: Use this line for High Sierra
   -monitor stdio
   -device vmware-svga
+  # -spice port=5900,addr=127.0.0.1,disable-ticketing=on
```

Comment:
:+1:

Reply:
@ebenoist thanks for the comment, do you want me to do something more here? :)

Reply:
Nope. I just think it's a great idea. I searched for similar settings when getting things running. I'm using virt-manager now to manage all of that, but it would have been helpful for setup.

Reply:
ok great, so resolving :)